src/app/zap-templates/templates/app/attribute-accessors.zapt

```diff
@@ -0,0 +1,62 @@
+{{>header}}
+
+/**
+ *  @file
+ *    This file contains declarations for accessors around clusters attributes.
+ */
+
+#pragma once
+
+#include <app/util/af-types.h>
+#include <lib/support/Span.h>
+
+namespace chip {
+namespace app {
+{{#zcl_clusters}}
+
+namespace {{asCamelCased label false}} {
+/*
+ * Cluster Attributes Accessors for: {{label}}
+ *
+ * Side: Server
+{{#zcl_attributes_server}}
+{{#if clusterRef}}
+ *   - {{label}}
+{{/if}}
+{{/zcl_attributes_server}}
+ *
+ * Side: Client
+{{#zcl_attributes_client}}
+{{#if clusterRef}}
+ *   - {{label}}
+{{/if}}
+{{/zcl_attributes_client}}
+ *
+ */
+
+namespace Server {
+
+{{#zcl_attributes_server}}
+{{#if clusterRef}}
+EmberAfStatus Get{{asCamelCased label false}}(chip::EndpointId endpoint, {{#if (isOctetString type)}}chip::ByteSpan{{else}}{{asUnderlyingZclType type}}{{/if}} * {{asCamelCased label}});
```

Comment:
Is there a reason not to use `asChipUnderlyingType` here and elsewhere in this PR?  Would be a lot clearer....